### ISSUE TITLE
RandomErasing with same_on_batch=True

### ISSUE THREAD

**Issue by TropComplique** [kornia/kornia#822]
## 🐛 Bug

RandomErasing is not the same on a batch when same_on_batch is True.

## To Reproduce (in Jupyter Notebook)

```python
import numpy as np
import torch

from kornia.augmentation import RandomErasing
from PIL import Image

x = Image.open('image.jpg')
x = torch.from_numpy(np.array(x)).float().div(255.0)
x = torch.stack(5 * [x]).permute(0, 3, 1, 2)
# it has shape [5, 3, h, w]

erase = RandomErasing(same_on_batch=True, p=1.0)
x = erase(x)

x = torch.cat(x.chunk(5, 0), dim=3)
# it has shape [1, 3, h, 5 * w]

Image.fromarray(x[0].permute(1, 2, 0).mul(255.0).byte().cpu().numpy())
```

## Expected behavior

Erasing must be the same on each image in the batch.

The current output is:
![example](https://user-images.githubusercontent.com/19353433/102777498-59c3e600-4399-11eb-8238-16cb412e0c7c.png)
## Environment

torch==1.7.1
kornia==0.4.1

**Comment by shijianjian** [comment]
Hi, I have tested your code on the master branch and I cannot reproduce. Can you try install from master?

If the problem still happens, can you provide a specific seed to secure the reproducibility?

**Comment by shijianjian** [comment]
Closed due to no response.
